README.md

```diff
@@ -40,9 +40,12 @@ git clone git@github.com:angr/angr-management.git
 ```
 cd angr-management
 pip install -e .
+pip install -r requirements.txt
```

Comment:
This isn't necessary. It may have been before today but I fixed a typo in setup.py.